etstool.py

```diff
@@ -176,17 +175,15 @@ def install(edm, runtime, environment, editable, docs, source):
         "{edm} --config edm.yaml install -y -e {environment} " + packages,
         "{edm} plumbing remove-package -e {environment} traits",
     ]
-    if editable:
-        install_cmd = (
-            "{edm} run -e {environment} -- "
-            "python -m pip install --editable . --no-dependencies"
-        )
-    else:
-        install_cmd = (
-            "{edm} run -e {environment} -- "
-            "python -m pip install . --no-dependencies"
-        )
+
+    install_cmd = _get_install_command_string(".", editable)
+    install_mypy_cmd = _get_install_command_string("mypy", editable, False)
```

Comment:
If this catches on, it may make sense to request `mypy` in the buildsystem, so that we can EDM install it. I'll create an issue.

Reply:
> I'll create an issue.

See enthought/buildsystem#2984

Reply:
Thank you for creating the issue !